reactions/src/HALO.cpp

```diff
@@ -671,6 +715,147 @@ double HALO::plinear_cosmosis(double k) const {
 }
 
 
+// vars:
+// 0 = acol, 1= omega0, 2 = mg param
+void HALO::react_init_nu(double vars[]) const{
+  SPT spt(C, P_l, epsrel);
+
+  double pspt, psptp, p1h, p1hp, plm, plcb, plnu, psptcbv, argument;
+
+  // fv
+  double fv  = vars[1]/vars[7];
+  double fv2 = pow(fv,2);
+  double fvt = 1.-fv;
+  double fvt2  = pow(fvt,2);
+
+/// mathcal E///
+  bigE = pow2(1.-fv)*one_halo(0.01,vars)/one_halop(0.01,vars);
```

Comment:
These 0.01 and 0.06 k values, why these numbers and wouldn't it be better if they were named variables?

Reply:
Agree. Have added them as variables with a comment and reference.